lib/cnb/daily_rates.rb

```diff
@@ -1,10 +1,17 @@
+require 'date'
+
 module CNB
-  class DailyRates < CurrencyRates
-    def initialize
-      @currencies = {}
-      @cur_rates_url = "#{CUR_RATES_URL}kurzy_devizoveho_trhu/denni_kurz.xml"
+  class DailyRates < Base
+    def rate_for(currency, date = Date.today)
+      normalized_date = normalize_date(date)
+      fail DateTooOld if max_past(normalized_date, CNB::DAILY_MAX_PAST)
```

Comment:
pro tohle mi prijde zbytecna metoda max_past, zaprve to bych spis cekal, ze to bude vracet podle nazvu nejaky max datum, zadruhe to je jen jedna radka v ty metode, za treti tam musime posilat konstantu stejne jako argument. Myslim, ze kdyz to porovnani bude tady, tak to bude citelnejsi i